### PR TITLE
Webserver

### PR DESCRIPTION
**Pull request by abake-enova**
Okay, I finally decided to push up what I've got for the webserver. I'm new to Django, so bear with me if you see anything atrocious. Currently, it supports creating, joining, and starting games, as well as passing priority. But I think I'll need to talk with you about how to hand game_state responses since I was having some trouble figuring out whose turn it is.

### CODE REVIEW

File: clients/webserver/deckr/deckr/urls.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Add an import:  from blog import urls as blog_urls
+    2. Add a URL to urlpatterns:  url(r'^blog/', include(blog_urls))
+"""
```
Comment (courageousillumination) [inline clients/webserver/deckr/deckr/urls.py]: Can you remove the default doc string?

File: clients/webserver/deckr/webclient/static/proto/client_message.proto [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import 'game_action';
```
Comment (courageousillumination) [inline clients/webserver/deckr/webclient/static/proto/client_message.proto]: Can the proto files be symlinked or excluded? No real reason to have them in the repository twice.

File: clients/webserver/deckr/webclient/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (courageousillumination) [inline clients/webserver/deckr/webclient/tests.py]: No tests :(

**Comment by courageousillumination** [comment]
Looks like a pretty good start. active_player says who's turn it is on the game state (it's a game ID for the active player).

**Comment by courageousillumination** [comment]
I'll merge this in as long as we can address my comments in the near future.

File: clients/webserver/deckr/webclient/tests.py (hunk 3 above)
Comment (alexthebake) [inline clients/webserver/deckr/webclient/tests.py]: Yea.. I should do that. I did lettuce tests! d:

File: clients/webserver/deckr/webclient/static/proto/client_message.proto (hunk 2 above)
Comment (abake-enova) [inline clients/webserver/deckr/webclient/static/proto/client_message.proto]: Yea, I originally wanted to symlink them, but I was having trouble with the import statements. I'll see if I can fix this though. Also, I normally run the decker_server from whichever branch you are working on, so I was having some trouble with inconsistencies between proto files on the server and client, so I just need to be careful with that.

**Pull request merged**